Algorithms/Strings/NaiveStringSearch.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+
+namespace Algorithms.Strings
+{
+    /// <summary>
+    /// Implements the traditional naive string matching algorithm in C#.
+    /// Cost:  O(n*m), where m is pattern length and n is content length.
+    /// </summary>
+    public static class NaiveStringSearch
+    {
+      public static void NaiveSearch(String content, String pattern)
+      {
+          int m = pattern.Length;
+          int n = content.Length;
+
+          for (int e = 0; e <= (n - m); e++)
+          {
+            int j;
+
+            for (j = 0; j < m; j++)
+                {
+                  if (content[e + j] != pattern[j])
+                  break;
+                 }
+
+            if (j == m)
+                Console.WriteLine("Pattern occurs in content at index " + e);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=4766632)